src/blocks/TitleParagraphBlockBrandPivot.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  ContentWrapper,
+  MOBILE_BP_DOWN,
+  SectionWrapper,
+} from '../components/blockHelpers'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+} from './BaseBlockProps'
+
+const Headline = styled('h4')({
+  textTransform: 'uppercase',
+  fontSize: '1rem',
+  margin: 0,
+  [MOBILE_BP_DOWN]: {
+    textAlign: 'center',
+  },
+})
+
+const Paragraph = styled('div')({
+  display: 'block',
+  maxWidth: '100%',
+  width: '100%',
```

Comment:
You can remove all of the `width: 100% , maxWidth: 100% , display: block` since a div has `display: block` and `width: 100%` as default :)